dev-lang/mujs/mujs-0_p20161031.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils flag-o-matic toolchain-funcs vcs-snapshot
```

Comment:
No idea why it is there.  Just copied from older version.  How to find out if it's needed ?